<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>balanceScreen</class>
 <widget class="QMainWindow" name="balanceScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>604</width>
    <height>559</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>atm3</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(102, 102, 102);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="check_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>AVAIBLE BALANCE</string>
    </property>
   </widget>
   <widget class="QPushButton" name="return1_button">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>201</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(254, 204, 102);</string>
    </property>
    <property name="text">
     <string>RETURN MENU</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../gümüst/icons/reply.png</normaloff>
      <normalon>../../gümüst/icons/icns/reply.png</normalon>../gümüst/icons/reply.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="balance_label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>321</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(204, 204, 204);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>604</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
